inc/saihostintf.h

```diff
@@ -276,6 +276,10 @@ typedef enum _sai_hostif_trap_channel_t
 
 typedef enum _sai_hostif_trap_attr_t
 {
+    /** Host interface trap ID [sai_hostif_trap_id_t]
+     * (CREATE_ONLY|MANDATORY_ON_CREATE|KEY) */
+    SAI_HOSTIF_TRAP_ATTR_TRAP_ID,
```

Comment:
Calling this and the trap handle (hostif_trap_id) same might cause confusion. Please think about renaming the Enum as sai_hostif_trap_t/sai_hostif_trap_type_t and for eg SAI_HOSTIF_TRAP_ATTR_ID_VRRP ->SAI_HOSTIF_TRAP_ATTR_VRRP/SAI_HOSTIF_TRAP_ATTR_TYPE_VRRP.